language-support/ts/codegen/tests/ts/build-and-lint-test/src/__tests__/test.ts

```diff
@@ -582,6 +608,13 @@ test('party API', async () => {
 
 });
 
+
+test('user API', async () => {
```

Comment:
```suggestion
test('getUser', async () => {
```

Unless you intend to expand this test to test the entire surface of the API in subsequent PRs.

Reply:
I'll keep it for the other methods that I'm going to add